bfvmm/include/vmcs/vmcs_intel_x64_host_vm_state.h

```diff
@@ -109,77 +111,77 @@ class vmcs_intel_x64_host_vm_state : public vmcs_intel_x64_state
 
         bfdebug << bfendl;
         bfdebug << "segment selectors:" << bfendl;
-        PRINT_STATE(m_es);
-        PRINT_STATE(m_cs);
-        PRINT_STATE(m_ss);
-        PRINT_STATE(m_ds);
-        PRINT_STATE(m_fs);
-        PRINT_STATE(m_gs);
-        PRINT_STATE(m_ldtr);
-        PRINT_STATE(m_tr);
+        bfdebug << std::setw(35) << view_as_pointer(m_es) << bfendl;
```

Comment:
35 because everything fits with that width. It's easy to find/replace if we want to change the alignment to something else. It was actually there all along, all I did was remove the macro (as that's a general thing that will be happening in coming PRs before 1.1)